matchbox/models/utils.py

```diff
@@ -25,6 +23,10 @@ def generate_id():
     return uuid.uuid4().hex
 
 
+def generate_uuid():
```

Comment:
i don't see place where you use this function

Reply:
There is also a function generate_id() which is also not used anywhere. I believe the utils module should provide also some helpers for users, i.e. generate IDs. And I was using UUID in it's pure form, not the "hex" ones.